.travis.yml

```diff
@@ -19,6 +19,9 @@ matrix:
       env: TEST_SUITE=default-site
     - rvm: *ruby1
       env: ROUGE_VERSION=1.11.1 # runs everything with this version
+    - rvm: *ruby1
```

Comment:
crude benchmarks run on a test repo has shown that Jekyll's build times are considerably faster on Ruby 2.5 in comparison with older rubies..
The trend observed is..:
```
Ruby 2.5 < Ruby 2.4 < Ruby 2.3 < Ruby 2.2
```

Reply:
But is there a change that would make Jekyll _faster_ on one and _slower_ on another? Seems unlikely.
At least to start, it seems like we should be focusing our limited efforts on optimizing for performance with latest Ruby.

Reply:
The observations are that the time difference is greater on older rubies.. So, at times there is a chance that our decision to ship a change be wrongly influenced by a small improvement in build time on Ruby 2.5..

The observations are at https://travis-ci.org/ashmaroli/related-posts-test if you need a reference